install/kubernetes/cilium/templates/hubble-tls-secret.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if and .Values.hubble.enabled .Values.hubble.tls.enabled }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hubble-ca-cert
+  namespace: {{ .Release.Namespace }}
+data:
+{{- if and (.Values.global.hubble.tls.auto.enabled) (eq .Values.global.hubble.tls.auto.method "helm") }}
+{{ include "ca.gen-cert-only" . | indent 2 }}
+{{- else }}
+  ca.crt: |-
+{{ .Values.hubble.tls.ca.cert | b64dec | indent 4 -}}
+{{- end }}
+---
```

Comment:
I just tripped over this a little... I mean, the current version in master has 4 objects in `secret.yaml`, which doesn't make friends with the installation method I'm pursuing for OpenShift. Namely, `openshift-install` only reads the topmost object in a multi-doc YAML. Any chance we could stick to _one-object-per-file_ convention?

Reply:
done!

Reply:
thanks!